themes/admin.py

```diff
@@ -0,0 +1,87 @@
+from django.contrib import admin
+
+from django.contrib.auth.admin import UserAdmin
+from .models import (
+	Themes, Review, 
```

Comment:
don't do this. We don't write the codes so it looks good in the eyes of our fellow developers. We write codes in a way that it is efficient and understandable. follow PEP